source/components/cardContainer/card/headerColumn/sizeForBreakpoints.tests.ts

```diff
@@ -0,0 +1,74 @@
+import { SizeForBreakpoints } from './sizeForBreakpoints';
+
+import { services } from 'typescript-angular-utilities';
+import __string = services.string;
+import __object = services.object;
+
+describe('SizeForBreakpoints', () => {
+	let sizeForBreakpoints: SizeForBreakpoints;
+
+	beforeEach(() => {
+		sizeForBreakpoints = new SizeForBreakpoints(__string.stringUtility, __object.objectUtility);
+	});
+
+	it('should convert an object of breakpoint sizes into a class string', () => {
+		const sizes = {
+			lg: 4,
+			md: 3,
+			sm: 2,
+			xs: 1,
+		};
+		expect(sizeForBreakpoints.getClass(sizes, '')).to.equal('col-xs-1 col-sm-2 col-md-3 col-lg-4 ');
```

Comment:
A nice idea, but at what point does the view get too logic intensive? Particularly since views in general are un-testable or at least much more difficult to test.